torchdynamo/output_graph.py

```diff
@@ -433,3 +433,25 @@ def cleanup(self):
         for node in self.graph.nodes:
             if "example_value" in node.meta:
                 del node.meta["example_value"]
+
+    def create_proxy(
+        self,
+        kind,
+        target,
+        args,
+        kwargs,
+        name=None,
+        type_expr=None,
+        proxy_factory_fn=None,
+        current_tx=None,
+    ):
+        rv = super().create_proxy(
+            kind, target, args, kwargs, name, type_expr, proxy_factory_fn
+        )
+
+        # append stack trace to fx node
+        tx = current_tx if current_tx else self.root_tx
+        frame_summary = tx.frame_summary()
+        rv.node.stack_trace = f"{frame_summary}\n  {frame_summary.line}"
```

Comment:
If you want to print a `traceback.FrameSummary` you could use code like:
https://github.com/pytorch/torchdynamo/blob/9ecbdaed5c1afe2f8443e0f13e081191cc29b0d3/torchdynamo/exc.py#L55
which I'd expect to look a bit nicer.

The other option would be to leave it as a `traceback.FrameSummary` object, which will allow users to query file/line more easily.

Reply:
done.